src/main/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistant.java

```diff
@@ -111,15 +111,21 @@ public void editAssignmentInCanvas(Long courseId, AttendanceAssignment attendanc
     }
 
     public void deleteAssignmentInCanvas(Long canvasCourseId, OauthToken oauthToken) throws AttendanceAssignmentException{
-
         List<AttendanceSection> sections = attendanceSectionService.getSectionByCanvasCourseId(canvasCourseId);
         if(CollectionUtils.isEmpty(sections)) {
-            throw new AttendanceAssignmentException(AttendanceAssignmentException.Error.NON_EXISTENT_SECTION_ERROR);
+            LOG.warn("Sections empty");
         }
 
-        AttendanceAssignment assignment = assignmentRepository.findByAttendanceSection(sections.get(0));
+        AttendanceAssignment assignment = assignmentRepository.findByAttendanceSection(sections.get(0));;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)